kong/plugins/aws-lambda/schema.lua

```diff
@@ -1,3 +1,14 @@
+local function check_status(status)
+  if not status then
```

Comment:
Hmmm possible - it's been too long since I last touched this. If so, then I think the logic can be shortened and we can only use 1 branch, that returns `false` if `status and status isn't in range`, and `true` otherwise.